src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: 198px;
+  height: 406px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Add padding for all sides of the card